presto-docs/src/main/sphinx/release/release-342.md

```diff
@@ -0,0 +1,77 @@
+# Release 342 (XX Sep 2020)
+
+## General Changes
+
+* Add {func}`from_iso8601_timestamp_nanos` function. ({issue}`5048`)
+* Improve performance of queries that use the `DECIMAL` type. ({issue}`4886`)
+* Improve performance of queries involving `IN` with subqueries by extending support for dynamic
+  filtering and dynamic partition pruning. ({issue}`5017`)
```

Comment:
What does "partition pruning" mean in the context of the engine?

Reply:
Let's maybe use just `dynamic filtering` as an umbrella keyword.